src/php/Printer/TypePrinter/CallablePrinter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Printer\TypePrinter;
+
+/**
+ * @implements TypePrinterInterface<object>
+ */
+final class CallablePrinter implements TypePrinterInterface
+{
+    /**
+     * @param object $form
+     */
+    public function print($form): string
+    {
+        return '<function>';
```

Comment:
I choose `<function>` because we already have these:
- `'<PHP-Object(...)>'`
- `'<PHP-AnonymousClass>'`

What do you say? I was also considering `:function`, because `(type fn)` says
```clojure
phel:2> (type (fn [] "Hello World"))
:function
```

Reply:
`<function>` is better. `:function` is a keyword and that could be misleading. The following two lines would evaluate to the same result:

```
phel:1> (type (fn [] "Hello World"))
:function
phel:2> :function
:function
```

Reply:
I didn't think about it... now that you mention it, now I like to have `<function>`.